pkg/printer/printer.go

```diff
@@ -4,6 +4,15 @@ import (
 	"github.com/doitintl/kube-no-trouble/pkg/judge"
 )
 
+var Printers = map[string]Printer{
```

Comment:
Thinking about this more, maybe the solution is to have this map, but instead of storing the printers, we could store relevant functions to create those printers, and call once needed?

Reply:
Yeah thats what I wanted ^^, I couldn't find away to do it

Reply:
I wanted:

map[String]func -> Printer

Reply:
ok found a way to do it